### DISCUSSION TITLE
Can't recover my password as I don't have access to my university email any more.

### DISCUSSION THREAD

**Discussion by amiramirbn73** [community/community#147405] [GitHub Education]
Hi everyone,

I want to log in into my student account but I forgot my password. Also as my access to my uni email is terminated, I can't recover it. How can I solve this issue, please?!

**Comment by davevad93** [comment]
Hi @amiramirbn73 ,

Try contacting support by opening a ticket [here](https://support.github.com/contact/account), select "`Device verification on sign-in, unable to access email`" and explain your issue.
